### PR TITLE
[2.7] Fix issues found by PHPStan

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

This is the subpart of #25536 that applies on 2.7.
ping @dkarlovi FYI.

### CODE REVIEW

File: src/Symfony/Bridge/Doctrine/Tests/Form/Type/EntityTypeTest.php [hunk 1]
```diff
@@ -1103,8 +1102,7 @@ public function testLoaderCaching()
         $repo = $this->em->getRepository(self::SINGLE_IDENT_CLASS);
 
         $entityType = new EntityType(
-            $this->emRegistry,
-            PropertyAccess::createPropertyAccessor()
+            $this->emRegistry
```
Comment (ro0NL) [inline src/Symfony/Bridge/Doctrine/Tests/Form/Type/EntityTypeTest.php]: single line?

Comment (nicolas-grekas) [reply src/Symfony/Bridge/Doctrine/Tests/Form/Type/EntityTypeTest.php]: yep, fixed

**Review by ro0NL — APPROVED** [review]

**Review by javiereguiluz — APPROVED** [review]
Nice fixes. Wrong namespaces and wrong spelling of PHP functions is always hard to detect. Thanks!

**Comment by nicolas-grekas** [comment]
Thank you @dkarlovi.

**Comment by nicolas-grekas** [comment]
Now merged up to master.

**Comment by dkarlovi** [comment]
Nice. :+1:

**Pull request merged**